mods/d2k/maps/harkonnen-01b/harkonnen01b.lua

```diff
@@ -61,57 +61,57 @@ Messages =
 
 CachedResources = -1
 Tick = function()
-	if AtreidesArrived and atreides.HasNoRequiredUnits() then
-		player.MarkCompletedObjective(KillAtreides)
+	if AtreidesArrived and Atreides.HasNoRequiredUnits() then
+		Player.MarkCompletedObjective(KillAtreides)
 	end
 
-	if player.Resources > SpiceToHarvest - 1 then
-		player.MarkCompletedObjective(GatherSpice)
+	if Player.Resources > SpiceToHarvest - 1 then
+		Player.MarkCompletedObjective(GatherSpice)
 	end
 
 	-- player has no Wind Trap
-	if (player.PowerProvided <= 20 or player.PowerState ~= "Normal") and DateTime.GameTime % DateTime.Seconds(32) == 0 then
+	if (Player.PowerProvided <= 20 or Player.PowerState ~= "Normal") and DateTime.GameTime % DateTime.Seconds(32) == 0 then
 		HasPower = false
 		Media.DisplayMessage(Messages[2], Mentat)
 	else
 		HasPower = true
 	end
 
 	-- player has no Refinery and no Silos
-	if HasPower and player.ResourceCapacity == 0 and DateTime.GameTime % DateTime.Seconds(32) == 0 then
+	if HasPower and Player.ResourceCapacity == 0 and DateTime.GameTime % DateTime.Seconds(32) == 0 then
 		Media.DisplayMessage(Messages[3], Mentat)
 	end
 
-	if HasPower and player.Resources > player.ResourceCapacity * 0.8 and DateTime.GameTime % DateTime.Seconds(32) == 0 then
+	if HasPower and Player.Resources > Player.ResourceCapacity * 0.8 and DateTime.GameTime % DateTime.Seconds(32) == 0 then
 		Media.DisplayMessage(Messages[4], Mentat)
 	end
 
-	if player.Resources ~= CachedResources then
-		local parameters = { ["harvested"] = player.Resources, ["goal"] = SpiceToHarvest }
+	if Player.Resources ~= CachedResources then
+		local parameters = { ["harvested"] = Player.Resources, ["goal"] = SpiceToHarvest }
 		local harvestedResources = UserInterface.Translate("harvested-resources", parameters)
 		UserInterface.SetMissionText(harvestedResources)
-		CachedResources = player.Resources
+		CachedResources = Player.Resources
 	end
 end
 
 WorldLoaded = function()
-	player = Player.GetPlayer("Harkonnen")
-	atreides = Player.GetPlayer("Atreides")
+	Player = Player.GetPlayer("Harkonnen")
```

Comment:
This will cause errors because `Player` is shadowing the global table used for `Player.GetPlayer`. If you want to change this it should be `player` -> `Harkonnen`.